### ISSUE TITLE
SiSyPHuS BSI Windows 10 Telemetriedaten

### ISSUE THREAD

**Issue by Tealk** [crazy-max/WindowsSpyBlocker#123]
The "[Bundesamt für Sicherheit in der Informationstechnik](https://www.bsi.bund.de/DE/Themen/Cyber-Sicherheit/Empfehlungen/SiSyPHuS_Win10/AP4/SiSyPHuS_AP4.html)" has found the following telemetry components.
You could record them here?

```
0.0.0.0 oca.telemetry.microsft.com
0.0.0.0 alpha.telemetry.microsft.com
0.0.0.0 vortex-win-sandbox.data.microsoft.com
0.0.0.0 eu.vortex-win.data.microsft.com
0.0.0.0 us.vortex-win.data.microsft.com
0.0.0.0 v10.vortex-win.data.microsft.com
0.0.0.0 geo.vortex.data.microsoft.com.akadns.net
0.0.0.0 v10-win.vortex.data.microsft.com.akadns.net
0.0.0.0 db5.vortex.data.microsoft.com.akadns.net
0.0.0.0 asimov-win.settings.data.microsoft.com.akadns.net
0.0.0.0 db5.settings-win.data.microsoft.com.akadns.net
0.0.0.0 settings-win.data.microsoft.com
0.0.0.0 db5-eap.settings-win.data.microsoft.com.akadns.net
0.0.0.0 geo.settings-win.data.microsoft.com.akadns.net
```

**Comment by rafuse** [comment]
More info can be found here:
https://www.ghacks.net/2018/11/23/german-federal-office-bsi-publishes-telemetry-analysis/

**Comment by crazy-max** [comment]
Half are already present in the current version and the next version already contains new IPs and FQDNs shown by this report following the migration of my virtual machines to 1809 17763.134 release.
Too bad the report doesn't specify the exact release of Windows btw.
Thanks for your input, I will link to this report in the Wiki.

**Comment by crazy-max** [comment]
Solved in 4.18.1

**Comment by Ainatar** [comment]
Hi, there are plenty of "microsft" typos on this release:

alpha.telemetry.microsft.com
eu.vortex-win.data.microsft.com
us.vortex-win.data.microsft.com
v10-win.vortex.data.microsft.com.akadns.net

**Comment by crazy-max** [comment]
@Ainatar Fixed in 4.18.2

**Comment by F3000** [comment]
Does BSI have also a version for 1809 ?  So your list can help governmental organizations to get a better acceptance for windows 10 ?
